site/src/sphinx/server-annotated-service.rst

```diff
@@ -132,6 +132,29 @@ you can use :api:`@StatusCode` annotation as follows.
         public void deleteUser(@Param("name") String name) { ... }
     }
 
+A service method can receive a request only if it contains a header or parameter the method requires.
```

Comment:
Not sure this is better but how about `You can define a service method which handles a request only if it ...`

Reply:
Sounds better. Let me fix it. :-)